layer1/monitor/monitor.go

```diff
@@ -393,6 +399,7 @@ type logWork struct {
 	err       error
 }
 
+// eventSorter is an internal struct used to sort the keep track of processing events
```

Comment:
Internal struct is implied.

```suggestion
// eventSorter is used to sort the keep track of processing events.
```